### PR TITLE
Use os.path.normpath to make sure we use the correct os path (fixes t…

### PR DESCRIPTION
**Pull request by bellenot**
…he generation of allHeaders.h on Windows)

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/vc15 with flags -Droofit=OFF

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/vc15 with flags -Dall=OFF -Droofit=OFF

File: build/unix/makepchinput.py [hunk 1]
```diff
@@ -230,37 +230,37 @@ def isDirForPCH(dirName):
...
-                           "math/foam",
-                           "math/fumili",
-                           "math/mlp",
-                           "math/quadp",
-                           "math/rtools"
-                           "math/splot",
-                           "math/unuran",
-                           "math/vdt",
-                           "tmva/rmva")
+   PCHPatternsWhitelist = (os.path.normpath("interpreter/"),
```
Comment (Axel-Naumann) [inline build/unix/makepchinput.py]: Can you run `os.path.normpath` in `isAnyPatternInString()`? Seems a lot less verbose...

Comment (bellenot) [reply build/unix/makepchinput.py]: Good point! I'll try and commit the changes

**Comment by bellenot** [comment]
@phsft-bot build also on centos7/gcc49, mac1012/native, slc6/gcc62, ubuntu14/native

File: build/unix/makepchinput.py [hunk 2]
```diff
@@ -256,7 +256,7 @@ def isDirForPCH(dirName):
                            "math/fumili",
                            "math/mlp",
                            "math/quadp",
-                           "math/rtools"
+                           "math/rtools",
```
Comment (Axel-Naumann) [inline build/unix/makepchinput.py]: Yikes, well spotted!

**Pull request merged**